docs/workflows/sdk-setup.md

```diff
@@ -0,0 +1,291 @@
+---
+title: SDK Setup
+sidebar_position: 8
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+The Embedded SDK is a holistic SDK solution offering the entire experience embedded in your product. Users will not need to download the Beyond Identity Authenticator. A set of functions are provided to you through the `Embedded` namespace. This SDK supports OIDC and OAuth2.
+
+## Sample App
+
+<Tabs groupId="sdks">
+<TabItem value="android" label="Android">
+
+:::tip Sample App
+Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-android/tree/main/app) for the Embedded SDK.
+:::
+
+</TabItem>
+<TabItem value="ios" label="iOS">
+
+:::tip Sample App
+Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-swift/tree/main/Example) for the Embedded SDK.
+:::
+
+</TabItem>
+<TabItem value="javascript" label="JavaScript">
+
+:::tip Sample App
+Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-js/tree/main/example) for the Embedded SDK.
+:::
+
+</TabItem>
+<TabItem value="flutter" label="Flutter">
+
+:::tip Sample App
+Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-flutter/tree/main/example) for the Embedded SDK.
+:::
+
+</TabItem>
+<TabItem value="reactnative" label="React Native">
+
+:::tip Sample App
+Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-react-native/tree/main/example) for the Embedded SDK.
+:::
+
+</TabItem>
+</Tabs>
+
+## Installation
+
+<Tabs groupId="sdks">
+<TabItem value="android" label="Android">
+
+### Gradle
+
+To enable the retrieval of Cloudsmith hosted packages via Gradle, we need to add the Cloudsmith repository to
+the `root/build.gradle` file.
+
+```groovy
+repositories {
+    maven {
+        url "https://packages.beyondidentity.com/public/bi-sdk-android/maven/"
+    }
+}
+```
+
+After the repository is added, we can specify the Beyond Identity dependencies.
+
+```groovy
+dependencies {
+    implementation 'com.beyondidentity.android.sdk:embedded:[version]'
```

Comment:
It might be cool if there was a way to query the latest released version and put that in place of all the `[version]`. Not sure how difficult that would be.

Reply:
Interesting idea 🤔 I think that might be better as a followup.

Reply:
yea we can explore it later